ddtrace/internal/serverless/mini_agent.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import os
+from platform import python_version_tuple
```

Comment:
Very minor nit: 

We can import the python from this internal module. There is a (small) performance benefit of accessing platform information once. 

https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/internal/compat.py#L52

Reply:
fixed